tests/core/test_path_local.py

```diff
@@ -0,0 +1,112 @@
+from pathlib import Path
+
+from pytest import fixture, mark, raises
+
+from seedcase_sprout.core import (
+    path_properties,
+    path_readme,
+    path_resource,
+    path_resource_data,
+    path_resource_raw,
+    path_resource_raw_files,
+    path_resources,
+)
+from tests.core.directory_structure_setup import (
+    create_test_data_package,
+    create_test_resource_structure,
+)
+
+
+@fixture
+def tmp_package(tmp_path):
+    """Set up a test data package with two resources."""
+    create_test_data_package(tmp_path)
+
+    create_test_resource_structure(tmp_path, ["raw_file_1.csv.gz"])
+    create_test_resource_structure(tmp_path, ["raw_file_2.csv.gz", "raw_file_3.csv.gz"])
+
+    return tmp_path
+
+
+@mark.parametrize(
+    "function, expected_path",
+    [
+        (path_properties, "datapackage.json"),
+        (path_readme, "README.md"),
+    ],
+)
+def test_path_local_root_files_return_expected_path(
```

Comment:
```suggestion
def test_path_local_root_functions_return_expected_path(
```
Or am I misunderstanding?